File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-independent.cs
var groups = LoadGroups(); // List<List<Item>>
var groupIndex = 0;
while (groupIndex < groups.Count) {
    var items = groups[groupIndex];
    var itemIndex = 0;
    while (itemIndex < items.Count) {
        ProcessItem(items[itemIndex]);
        itemIndex = itemIndex + 1;
    }
    var notifications = LoadNotifications(groups[groupIndex]);
    var notificationIndex = 0;
    while (notificationIndex < notifications.Count) {
        SendNotification(notifications[notificationIndex]);
        notificationIndex = notificationIndex + 1;
    }
    groupIndex = groupIndex + 1;
}
ShowSummary();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-independent.php
<?php
$groups = loadGroups();
$groupIndex = 0;
while ($groupIndex < count($groups)) {
    $items = $groups[$groupIndex];
    $itemIndex = 0;
    while ($itemIndex < count($items)) {
        processItem($items[$itemIndex]);
        $itemIndex = $itemIndex + 1;
    }
    $notifications = loadNotifications($groups[$groupIndex]);
    $notificationIndex = 0;
    while ($notificationIndex < count($notifications)) {
        sendNotification($notifications[$notificationIndex]);
        $notificationIndex = $notificationIndex + 1;
    }
    $groupIndex = $groupIndex + 1;
}
showSummary();
